src/Common/xlf/CommonResources.it.xlf

```diff
@@ -4,42 +4,42 @@
     <body>
       <trans-unit id="AtLeastOneRepositoryHostIsRequired">
         <source>{0} item group is empty. At least one {1} repository host is required in order to generate SourceLink.</source>
-        <target state="new">{0} item group is empty. At least one {1} repository host is required in order to generate SourceLink.</target>
+        <target state="translated">Il gruppo di elementi {0} è vuoto. Per generare SourceLink, è necessario almeno un host di repository di {1}.</target>
         <note />
       </trans-unit>
       <trans-unit id="ItemOfItemGroupMustSpecifyMetadata">
         <source>Item '{0}' of item group '{1}' must specify metadata '{2}'</source>
-        <target state="new">Item '{0}' of item group '{1}' must specify metadata '{2}'</target>
+        <target state="translated">L'elemento '{0}' del gruppo di elementi '{1}' deve specificare i metadati '{2}'</target>
         <note />
       </trans-unit>
```

Comment:
This should perhaps be `il metadato` (singular) because `{2}` is only one metadata name (apparently always "Version") rather than a list. Likewise in other languages.